Generals/Code/GameEngine/Source/GameLogic/Object/Object.cpp

```diff
@@ -430,7 +430,12 @@ Object::Object( const ThingTemplate *tt, const ObjectStatusMaskType &objectStatu
 	m_soleHealingBenefactorID = INVALID_ID; ///< who is the only other object that can give me this non-stacking heal benefit?
 	m_soleHealingBenefactorExpirationFrame = 0; ///< on what frame can I accept healing (thus to switch) from a new benefactor
 
+	// TheSuperHackers @bugfix Mauller/xezon 02/08/2025 sendObjectCreated needs calling earlier before CreateModule's are initialized to prevent drawable related crashes
+	// We need to create the drawable early to prevent crashing when behavior modules try to initialize drawable data within thingFactory::newObject().
```

Comment:
First and second sentence now have overlapping explanations. This can perhaps be consolidated into one sentence?